apps/editing-toolkit/editing-toolkit-plugin/coming-soon/test/class-coming-soon-test.php

```diff
@@ -23,6 +35,13 @@ public static function tearDownAfterClass() {
 		parent::tearDownAfterClass();
 	}
 
+	/**
+	 * Post-test actions.
+	 */
+	public function tearDown() {
```

Comment:
```suggestion
	public function tearDown() {
		parent::tearDown();	
```

Need to call the parent `tearDown()`, otherwise mysterious bugs can appear 👻

Reply:
Thanks, I've added that call.